Tina-ttrimmis/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,50 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+let flickrObj = JSON.parse(jsonString);
+console.log(flickrObj);
+
+
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+let i;
```

Comment:
This works fine, but it is very common to just use the statement `let i = 0` as the first statement in parens after the `for` keyword declaring and assigning the variable all at once..